simple-rest-server/get.go

```diff
@@ -3,12 +3,40 @@ package main
 import (
 	"fmt"
 	"net/http"
-
+    "os"
+	"encoding/csv"
 	"github.com/julienschmidt/httprouter"
 )
 
 func ListTask(respWriter http.ResponseWriter, request *http.Request, params httprouter.Params) {
-	fmt.Fprintf(respWriter, "Show Details for ToDo")
+	 csvfile, err := os.Open("tasks.csv")
+
+         if err != nil {
+                 fmt.Println(err)
+                 return
+         }
+
+         defer csvfile.Close()
+
+         reader := csv.NewReader(csvfile)
+
+         reader.FieldsPerRecord = -1 
+         
+         rawCSVdata, err := reader.ReadAll()
+
+         if err != nil {
+                 fmt.Println(err)
+                 os.Exit(1)
+         }
+
+         // sanity check, display to standard output
+         for _, each := range rawCSVdata {
+             status := "unchecked"
+             if each[1] == "true" {  // each[1] will be the status - either true or false
```

Comment:
In the CSV file I think we should use 0 for false and 1 for true. (Check with Olivier on what he is going to do)
If this is the case - then we should be able to make the call:
if each[1] 
rather than == "true"

Eitherway - if we are going to do string comparison we should use the build-in strings.Compare() function.